oxytcmri/interface/repositories/nifti_folders_mri_exam_repository.py

```
@@ -1,35 +1,107 @@
+from dataclasses import dataclass
 from pathlib import Path
-from typing import List, Optional
+from typing import Protocol, Dict, Callable, Optional, List, cast
```

Ruff (0.11.9)

`typing.Dict` imported but unused

Remove unused import

(F401)